examples/codepen.js

```diff
@@ -0,0 +1,91 @@
+const { example } = require("../src/helpers");
+
+const box = (key) => ({
+  box: `#box-${key}`,
+  get pre() {
+    return `${this.box} pre[role="presentation"]`;
+  },
+  get input() {
+    return `${this.box} textarea`;
+  },
+});
+
+const selectors = {
+  html: box("html"),
+  css: box("css"),
+  js: box("js"),
+};
```

Comment:
this is a really interesting pattern. i like it

Reply:
I'm not sure it's generally applicable but seemed handy to ensure consistent selectors so there's less to update when codepen changes.